tests/console/yast2_nfs_client.pm

```diff
@@ -60,6 +60,9 @@ sub run {
     # From now we need needles
     select_console 'root-console';
 
+    # Call conditionally systemd-vconsole-setup before accessing virtual consoles to avoid wrong visualization with ncurses
+    assert_script_run '/usr/lib/systemd/systemd-vconsole-setup';
```

Comment:
Thanks for your comments, 50VR is created and let see what's happen.

Reply:
Looks good, but please remove the word "conditionally" from the comment then, and maybe we could add a reference to https://bugzilla.suse.com/show_bug.cgi?id=1183234#c52.

Reply:
please, check the scope of the ticket, we need to call it conditionally because it doesn't happen in other products other than SLE 12 SP5.

Reply:
the cheapest way to do it is to just call this condition and write some comment with the bug, but the best way to do it is with a soft-failure, so if those characters are detected with a needle when opening the module, we close the module, run this workaround (we have a workarounds.pm file to store it) and continue the test. In that way you can also see when in happens, because maybe your 50 runs never triggered and only in some builds for some race conditions happens. Later on it is easy to remove the workaround in that way.

Reply:
@jknphy 

-  According to bug reports, the bug occurs when systemd-vconsole-setup was not run before reaching multi-user mode (login),  so instead of checking a needle we can check that the service is started (systemctl is-active systemd-vconsole-setup) as suggested earlier. Like this, we could at least confirm that this is the cause of the bug.
- Normally (requires VRs) running systemd-vconsole-setup is harmless, can be run anywhere at anytime. Problems occur when i did NOT run. So if we don't go for a soft-failure, it should be fine to just update the comment and run the script every time.

Reply:
> According to bug reports, the bug occurs when systemd-vconsole-setup was not run before reaching multi-user mode (login), so instead of checking a needle we can check that the service is started (systemctl is-active systemd-vconsole-setup) as suggested earlier. Like this, we could at least confirm that this is the cause of the bug.

The service is run and the binary also triggered by udev multiple times, the bug is a race condition which can't be checked for easily.